src/main/java/eu/lucaventuri/common/SystemUtils.java

```diff
@@ -226,8 +229,7 @@ public static long printTime(Runnable run, String description) {
         run.run();
 
         long time = System.currentTimeMillis() - start;
-
-        System.out.println(description + " : " + time + " ms");
+        logger.log(Level.FINE, description + " : " + time + " ms");
```

Comment:
This method is called printTime(), I think it should always print and it is unnecessary to log.